flower_shop/__manifest__.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "Sally Flower Shop",
+    'summary': """Sally's Flower shop contains many types of flowers""",
+    'description': """ 
+         Sally's Flower Shop - 
+         Sally runs a flower shop. She wants to use Odoo in order to run her daily tasks smoothly. 
+         This store grows its own flowers in a garden nearby. These flowers have a set of different attributes.
+        """,
+
+    'author': "HalimaHamood",
+    'website': "https://www.yourcompany.com",
+    'category': 'Uncategorized',
+    'version': '0.1',
+    'depends': ['base', 'sale', 'mail', 'website', 'website_sale', 'stock'],
```

Comment:
TIP: you don't need to import base, sale, website, or mail as  they are all dependency for website_sale